Server/tools/itemSets.json

```diff
@@ -0,0 +1,19 @@
+{"buyValueId":494,"cost":650,"counterId":0,"currency":12,"effectiveFrom":"0001-01-01T00:00:00","effectiveUntil":"0001-01-01T00:00:00","id":233,"name":"cs_skeleton_outfit_b","sellValueId":493,"thingForSaleId":14,"thingForSaleType":"itemSetValue","itemCategoryId":0,"genders":""},
+{"buyValueId":610,"cost":10,"counterId":0,"currency":12,"effectiveFrom":"0001-01-01T00:00:00","effectiveUntil":"0001-01-01T00:00:00","id":291,"name":"cs_sm_cammo_desert_b","sellValueId":609,"thingForSaleId":22,"thingForSaleType":"itemSetValue","itemCategoryId":0,"genders":""},
+{"buyValueId":612,"cost":10,"counterId":0,"currency":12,"effectiveFrom":"0001-01-01T00:00:00","effectiveUntil":"0001-01-01T00:00:00","id":292,"name":"cs_sm_cammo_green_b","sellValueId":611,"thingForSaleId":23,"thingForSaleType":"itemSetValue","itemCategoryId":0,"genders":""},
+{"buyValueId":614,"cost":10,"counterId":0,"currency":12,"effectiveFrom":"0001-01-01T00:00:00","effectiveUntil":"0001-01-01T00:00:00","id":293,"name":"cs_sm_electro_b","sellValueId":613,"thingForSaleId":24,"thingForSaleType":"itemSetValue","itemCategoryId":0,"genders":""},
+{"buyValueId":616,"cost":10,"counterId":0,"currency":12,"effectiveFrom":"0001-01-01T00:00:00","effectiveUntil":"0001-01-01T00:00:00","id":294,"name":"cs_sm_gold_b","sellValueId":615,"thingForSaleId":25,"thingForSaleType":"itemSetValue","itemCategoryId":0,"genders":""},
+{"buyValueId":618,"cost":10,"counterId":0,"currency":12,"effectiveFrom":"0001-01-01T00:00:00","effectiveUntil":"0001-01-01T00:00:00","id":295,"name":"cs_sm_olive_b","sellValueId":617,"thingForSaleId":26,"thingForSaleType":"itemSetValue","itemCategoryId":0,"genders":""},
+{"buyValueId":620,"cost":10,"counterId":0,"currency":12,"effectiveFrom":"0001-01-01T00:00:00","effectiveUntil":"0001-01-01T00:00:00","id":296,"name":"cs_sm_red_b","sellValueId":619,"thingForSaleId":27,"thingForSaleType":"itemSetValue","itemCategoryId":0,"genders":""},
+{"buyValueId":622,"cost":10,"counterId":0,"currency":12,"effectiveFrom":"0001-01-01T00:00:00","effectiveUntil":"0001-01-01T00:00:00","id":297,"name":"cs_sm_shadow_b","sellValueId":621,"thingForSaleId":28,"thingForSaleType":"itemSetValue","itemCategoryId":0,"genders":""},
+{"buyValueId":624,"cost":10,"counterId":0,"currency":12,"effectiveFrom":"0001-01-01T00:00:00","effectiveUntil":"0001-01-01T00:00:00","id":298,"name":"cs_sm_steel_b","sellValueId":623,"thingForSaleId":29,"thingForSaleType":"itemSetValue","itemCategoryId":0,"genders":""},
+{"buyValueId":795,"cost":100,"counterId":0,"currency":12,"effectiveFrom":"0001-01-01T00:00:00","effectiveUntil":"0001-01-01T00:00:00","id":378,"name":"cs_ninja","sellValueId":794,"thingForSaleId":5,"thingForSaleType":"itemSetValue","itemCategoryId":0,"genders":""}
```

Comment:
These aren't being used right now, do you plan to amend this PR with the change to the tool script or is that for the future?